src/api/mapping.ts

```diff
@@ -16,6 +16,9 @@ const getNextPort = (map, start=3002):number =>{
 
 mappingRouter.post('/', (req, res) => {
   const domainKeys = getMappings()
+  if(parseInt(req.body.port) < 3001){
+    return res.status(400)
```

Comment:
```
return res.status(400).json({message: "Port cannot be smaller than 3001"})
```

Reply:
Lastly, when you receive this response from the user, could you fire off an alert?